exchange/order/limits/storage.go

```diff
@@ -0,0 +1,118 @@
+package limits
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/thrasher-corp/gocryptotrader/common/key"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/order"
+)
+
+// LoadLimits loads all limits into private limit holder
+func LoadLimits(levels []MinMaxLevel) error {
+	return executionLimitsManager.loadLimits(levels)
+}
+
+// GetOrderExecutionLimits returns the order limit matching the key
+func GetOrderExecutionLimits(k key.ExchangePairAsset) (MinMaxLevel, error) {
+	return executionLimitsManager.getOrderExecutionLimits(k)
+}
+
+// CheckOrderExecutionLimits is a convenience method to check if the price and amount conforms
+// to the exchange order limits
+func CheckOrderExecutionLimits(k key.ExchangePairAsset, price, amount float64, orderType order.Type) error {
+	return executionLimitsManager.checkOrderExecutionLimits(k, price, amount, orderType)
+}
+
+func (e *executionLimits) loadLimits(levels []MinMaxLevel) error {
+	if len(levels) == 0 {
+		return ErrCannotLoadLimit
+	}
+	e.mtx.Lock()
+	defer e.mtx.Unlock()
+	if e.epaLimits == nil {
+		e.epaLimits = make(map[key.ExchangePairAsset]*MinMaxLevel)
+	}
+
+	for x := range levels {
+		if levels[x].Key.Exchange == "" {
+			return fmt.Errorf("cannot load levels for '%s' '%s': %w", levels[x].Key.Asset, levels[x].Key.Pair(), errExchangeNameEmpty)
+		}
+		if !levels[x].Key.Asset.IsValid() {
+			return fmt.Errorf("cannot load levels for '%s' '%s': %w", levels[x].Key.Exchange, levels[x].Key.Pair(), errAssetInvalid)
+		}
+		if levels[x].Key.Pair().IsEmpty() {
+			return fmt.Errorf("cannot load levels for '%s' '%s': %w", levels[x].Key.Exchange, levels[x].Key.Asset, errPairNotSet)
+		}
+		if levels[x].MinPrice > 0 &&
+			levels[x].MaxPrice > 0 &&
+			levels[x].MinPrice > levels[x].MaxPrice {
+			return fmt.Errorf("%w for %s %s %s supplied min: %f max: %f",
```

Comment:
Just noticed that we're not being consistent with quote usage like the other nits where we do use quotes for here and the below errors